types/relay-runtime/lib/network/RelayNetworkTypes.d.ts

```diff
@@ -111,3 +111,25 @@ export type Uploadable = File | Blob;
 export interface UploadableMap {
     [key: string]: Uploadable;
 }
+
+/**
+ * React Flight tree created on the server.
+ */
+export type ReactFlightServerTree = any;
+export type ReactFlightPayloadQuery = {
```

Comment:
Looks like Flow marks all these properties as `readonly`.  Do we wanna do that for TypeScript types too?

Reply:
oh, done in later commit :+1: